lib/Jsrt/ChakraCommon.h

```diff
@@ -2378,6 +2378,30 @@ typedef unsigned short uint16_t;
             _In_ JsPromiseContinuationCallback promiseContinuationCallback,
             _In_opt_ void *callbackState);
 
+    /// <summary>
+    ///     Creates a new JavaScript Promise object.
+    /// </summary>
+    /// <remarks>
+    ///     <para>
+    ///     Requires an active script context.
+    ///     </para>
+    /// </remarks>
+    /// <param name="promise">The new Promise object.</param>
```

Comment:
We're inconsistent about this elsewhere in this file as well. I wonder if it has to do with line length and VS doing something automatically with formatting?